content/docs/atom-guide/dev-guide/materials/lua-material-functor-api.md

```diff
@@ -6,13 +6,7 @@ toc: true
 
 Lua material functors allow custom logic for processing material properties. They can read property values, set shader inputs and shader options, configure render states, adjust material property visibility in the editor, and more.
 
-For general information on the Lua programming language, see [Learning Lua](docs/user-guide/scripting/lua/#learning-lua).
-
-{{< note >}}
-*Lua material functors* are distinct from *[Lua gameplay scripts](docs/user-guide/scripting/lua/)*. While they both use the Lua language, the APIs and programming environment are mostly separate. Features like EBuses, the Properties Table, and Entity integration are exclusive to gameplay scripts, and are not available to Lua material functors. The [Lua Math Library](/docs/user-guide/scripting/lua/math-library/) is the only part of the API that is common to both systems.
-{{< /note >}}
-
-## Name Context
+## Name context
```

Comment:
This merge accidentally stomps a section I added. I'll fix it.

Reply:
Ah, sorry about that!